src/main/java/net/smartcosmos/edge/things/domain/things/RestUnpagedThingResponse.java

```diff
@@ -0,0 +1,27 @@
+package net.smartcosmos.edge.things.domain.things;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Data
+@NoArgsConstructor
+@Builder
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties({ "version" })
+public class RestUnpagedThingResponse {
```

Comment:
Why have two models? The core service doesn't consider paging -- why should the edge service be different in this regard?

When we discussed the endpoint for the core service we decided the URNs in the request are sufficient to define the size of the response, and there is no need for paging.